ui-v2/tests/utils/handlers.ts

```diff
@@ -1,96 +1,91 @@
 import { http, HttpResponse } from "msw";
 
+export const prefectURL = (path: string) => {
```

Comment:
https://github.com/mswjs/msw/issues/397, but because we're including `/api` to the end, just doing string concat because it doesn't work as a `new URL` object

Reply:
I'd prefer if this function's name started with a verb. Here's a suggestion, but I'm open to alternatives.
```suggestion
export const buildApiUrl = (path: string) => {
```